packages/trader/src/Modules/MT5/Components/mt5-poa.jsx

```diff
@@ -178,15 +180,24 @@ class MT5POA extends React.Component {
     };
 
     componentDidMount() {
+        this.is_mounted = true;
```

Comment:
I think we have the `makeCancellablePromise` that we are using for this now?

Reply:
That's only for *AccountWizard* i believe? The *MT5 Financial STP Signup Modal* isn't derived from *AccountWizard* since it's a Trader component.

Reply:
Using that one we don't have to rely on the is_mounted, but either way is fine I guess. Once it's a functional component we can perhaps use a custom `useIsMounted()` hook

Reply:
Yep, planning to refactor the components to functional components with hooks as indicated [here](https://github.com/binary-com/deriv-app/pull/1781/files#diff-fb2df603c57827d62cb223b0bce9aef5R34)